doc/user-guide/user-guide.rst

```diff
@@ -325,3 +325,23 @@ Generic Camera Operations
 =========================
 
 TBD
+
+Generic Calsys Operations
+=========================
+
+Each telescope will have a calibration system (Calsys) that includes all components necessary to perform flats. This is accomplished by projecting light onto the calibration screen. Each system has the ability to perform two types of calibrations (`calib_type`): "white light" flats, which have a broadband illumination, and "monochromatic, which have a much narrower bandwidth of illumination. In the case of the the `maintel`, there is a separate broadband illumination source per filter band.  
```

Comment:
Please, make sure you use "semantic line break". Basically, have each phrase in a separate line.

Reply:
Right, thanks!